newIDE/app/src/Export/ExportLauncher.js

```diff
@@ -113,94 +141,100 @@ export default class ExportLauncher extends Component<Props, State> {
           errorId: 'export-error',
         });
       }
-
-      throw err;
     };
 
-    const exportPipelineContext = {
-      project,
-      updateStepProgress: this._updateStepProgress,
-      exportState: this.state.exportState,
-    };
+    const setStep = (step: BuildStep) => this.setState({ exportStep: step });
 
-    this.setState({
-      exportStep: 'export',
-      stepCurrentProgress: 0,
-      stepMaxProgress: 0,
-      errored: false,
-      build: null,
-    });
-    exportPipeline
-      .prepareExporter(exportPipelineContext)
-      .then(preparedExporter => {
-        return exportPipeline.launchExport(
-          exportPipelineContext,
-          preparedExporter
-        );
-      }, handleError(t('Error while preparing the exporter.')))
-      .then(exportOutput => {
-        this.setState({
-          exportStep: 'resources-download',
-        });
-        return exportPipeline.launchResourcesDownload(
-          exportPipelineContext,
-          exportOutput
-        );
-      }, handleError(t('Error while exporting the game.')))
-      .then(resourcesDownloadOutput => {
-        this.setState({
-          exportStep: 'compress',
-        });
-        return exportPipeline.launchCompression(
-          exportPipelineContext,
-          resourcesDownloadOutput
-        );
-      }, handleError(t('Error while downloading the game resources. Check your internet connection and that all resources of the game are valid in the Resources editor.')))
-      .then(compressionOutput => {
-        const { launchUpload, launchOnlineBuild } = exportPipeline;
-        if (!!launchUpload && !!launchOnlineBuild) {
-          this.setState({
-            exportStep: 'upload',
+    try {
+      setStep('register');
+      const profile = authenticatedUser.profile;
+      const getAuthorizationHeader = authenticatedUser.getAuthorizationHeader;
+      const gameId = project.getProjectUuid();
+      const authorName = project.getAuthor() || 'Unspecified author';
+      const gameName = project.getName() || 'Untitled game';
+      if (profile) {
```

Comment:
We could even run this logic in a separate function async that we dont await - in the sense that we launch it and immediately launch the export step (so there is no "register" step).
This could avoid a few hundreds milliseconds that should in theory not be waited for when building a game :)

Reply:
Yeah I thought about that and ended up waiting for it because it wasn't too long. 
I'll do it to see if it works well 👍